samples/language/RepeatUntilLoops.qs

```diff
@@ -0,0 +1,38 @@
+/// # Sample
+/// Repeat-Until Loops
+///
+/// # Description
+/// The repeat loop runs a block of code before evaluating a condition.
```

Comment:
nit: if we are referring to this as a "repeat-until loop", it should always be consistent, i.e. this sentence should be "The repeat loop runs a block of code before evaluating a condition.".

Reply:
done